libraries/botbuilder-dialogs-adaptive-runtime/src/coreBot.ts

```diff
@@ -48,16 +48,12 @@ export class CoreBot extends ActivityHandler {
         ResourceExtensions.useResourceExplorer(dialogManager, resourceExplorer);
         LanguageGeneratorExtensions.useLanguageGeneration(dialogManager);
         LanguageGeneratorExtensions.useLanguagePolicy(dialogManager, new LanguagePolicy(defaultLocale));
+
+        // Inserts skills dependencies into initial turn state
         SkillExtensions.useSkillClient(dialogManager, skillClient);
         SkillExtensions.useSkillConversationIdFactory(dialogManager, skillConversationIdFactory);
-        useTelemetry(dialogManager, botTelemetryClient);
 
-        /* TODO(jgummersall) reconcile:
```

Comment:
Verified that these are handled by the skill extensions stuff, and the `useBotState` in the `CoreBotAdapter`